### PR TITLE
Update dev-php/composer to 2.6.5

### PR DESCRIPTION
**Pull request by r7l**
This PR should update composer to the most recent version. It also update the dependencies as far as i can tell. Might not be perfectly sync with composer.json. It wasn't sync before and i couldn't see any issues so far. It's basically just minor versions not really matching.

When downloading Composer it didn't have any docs anymore. So i removed it from the Ebuild as it would fail otherwise.

### CODE REVIEW

File: dev-php/composer/composer-2.6.5.ebuild [hunk 1]
```diff
@@ -99,7 +99,4 @@ src_install() {
 	exeinto "/usr/share/${PN}/bin"
 	doexe "bin/${PN}"
 	dosym "../share/${PN}/bin/${PN}" "/usr/bin/${PN}"
-
-	dodoc CHANGELOG.md README.md doc/*.md
```
Comment (fcool) [inline dev-php/composer/composer-2.6.5.ebuild:103]: does the newer composer do not deliver documentation with it? Or why did you remove the documentation packaging?

File: dev-php/symfony-polyfill-mbstring/symfony-polyfill-mbstring-1.28.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+		--quiet \
+		--output autoload.php \
+		--template fedora2 \
+		--basedir . \
+		. \
+		|| die
+	echo "\Fedora\Autoloader\Dependencies::required([__DIR__ . '/bootstrap.php']);" >> autoload.php
+}
+
+# TODO: Check if README.md is installed twice
```
Comment (fcool) [inline dev-php/symfony-polyfill-mbstring/symfony-polyfill-mbstring-1.28.0.ebuild:35]: do you want me to check that? Or is it left over?
As this schmea with einstalldocs is repeated in other ebuilds, it might be you checked it elsewhere?

**Review by fcool — APPROVED** [review]

File: dev-php/composer/composer-2.6.5.ebuild (hunk 1 above)
Comment (r7l) [reply dev-php/composer/composer-2.6.5.ebuild:103]: As said in the PR comment: When downloading current version of Composer, those files are missing and causing errors. That's why i've removed them. I've not changed the URL for the download. It's the same as before just with the current version.

File: dev-php/symfony-polyfill-mbstring/symfony-polyfill-mbstring-1.28.0.ebuild (hunk 2 above)
Comment (r7l) [reply dev-php/symfony-polyfill-mbstring/symfony-polyfill-mbstring-1.28.0.ebuild:35]: It's a leftover and it's in multiple of the already existing Ebuilds. For that reason, i would consider this a topic on its own and leave it like it is for now. How about opening a ticket and work on it seperately?

File: dev-php/composer/composer-2.6.5.ebuild (hunk 1 above)
Comment (fcool) [reply dev-php/composer/composer-2.6.5.ebuild:103]: Ouch! I just realized, that I did not even read your comment. :( 
Sorry about that!

I'll check on this briefy - but I wouldn't consider the missing documentation as a complete show stopper. Of course it would be nice to have "something" around... but I guess everyone, needing information does not use the delivered documentation - I guess everyone prefers online resources for an online tool like compser ;)

File: dev-php/symfony-polyfill-mbstring/symfony-polyfill-mbstring-1.28.0.ebuild (hunk 2 above)
Comment (r7l) [reply dev-php/symfony-polyfill-mbstring/symfony-polyfill-mbstring-1.28.0.ebuild:35]: I've added the issue.
